Basket-Donation-Payments/Basket-Donation-Payments/Models/APIManager.h

```diff
@@ -18,12 +18,14 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (void)createPaymentIntentWithBasket:(Basket*)basket totalAmount:(NSNumber*)totalAmount withBlock:(void (^)(NSError *, NSDictionary *))completion;
 
-+ (void)submitPaymentWithCard:(STPPaymentMethodCardParams*)params clientSecret:(NSString*)clientSecret andBlock:(void (^)(NSError *, STPPaymentHandlerActionStatus))completion;
++ (void)submitPaymentWithCard:(STPPaymentMethodCardParams*)params clientSecret:(NSString*)clientSecret andBlock:(void (^)(NSError *, STPPaymentHandlerActionStatus, NSString *))completion;
 
 + (void) newStripeCustomerIdWithName:(NSString*)fullName andEmail:(NSString*)email withBlock:(void (^)(NSError *, NSString *))completion;
 
 + (void) newNonprofitConnectedAccountWithEmail:(NSString*)email withAuthorizationCode:(NSString*)code withBlock:(void (^)(NSError *, NSString *))completion;
 
++ (void)createTransfersWithAmount:(NSNumber*)amount toConnectedStripeAccs:(NSArray<NSString*>*)connectedStripeAccs withSourceTxId:(NSString*)sourceTxId withBlock:(void (^)(NSError *, NSString *))completion;
```

Comment:
Use `toConnectedStripeAccounts` instead of `toConnectedStripeAccs`. The convention in Objective-C is to not abbreviate parameter names in functions, or anything besides local variables really, even for local variables most people use full names except for things like index variables in a for loop.